inlong-manager/manager-web/src/main/resources/application-dev.properties

```diff
@@ -97,6 +97,11 @@ source.update.enabled=false
 source.update.before.seconds=60
 source.update.interval=60
 
+# If turned on, Regularly clear expired supplementary recording tasks
+subSource.cleansing.enabled=false
+subSource.cleansing.interval=10
```

Comment:
It's better to include time unit in the configuration name.

Reply:
Fixed.